pim-tool-back/src/main/java/vn/elca/training/repository/projectcontroller/ProjectRepository_Impl.java

```diff
@@ -0,0 +1,39 @@
+package vn.elca.training.repository.projectcontroller;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+import org.springframework.transaction.annotation.Transactional;
+import vn.elca.training.model.entity.Project;
+import vn.elca.training.model.exception.UserNotFoundException;
+
+import java.util.List;
+@Service
+public class ProjectRepository_Impl {
```

Comment:
[S] Please use Interface, take example as in code base https://github.com/dtuan39/elca-java-exercises/blob/main/pim-tool-back/src/main/java/vn/elca/training/service/ProjectService.java